app/src/main/java/org/mozilla/fenix/settings/logins/fragment/SavedLoginsAuthFragment.kt

```diff
@@ -74,10 +50,10 @@ class SavedLoginsAuthFragment : PreferenceFragmentCompat() {
      * so we are disabling the settings that navigate while authenticating.
      * https://github.com/mozilla-mobile/fenix/issues/12312
      */
-    private fun togglePrefsEnabledWhileAuthenticating(enabled: Boolean) {
-        requirePreference<Preference>(R.string.pref_key_password_sync_logins).isEnabled = enabled
-        requirePreference<Preference>(R.string.pref_key_save_logins_settings).isEnabled = enabled
-        requirePreference<Preference>(R.string.pref_key_saved_logins).isEnabled = enabled
+    private fun ensurePrefsEnabledWhileAuthenticating() {
```

Comment:
This looks like a bug. Originally we wanted these to be disabled while we were authenticating, and then enabled again once authentication either was successful or failed. Hence the toggling. 

It looks like somehow the disabling was accidentally refactored out at some point. We should either add it back in or remove this altogether if it's no longer needed.

Original bug:
See https://github.com/mozilla-mobile/fenix/issues/12312

Reply:
Nice catch! I was curious about this as well but I didn't find the diff where this call was removed.

I'll re-add the original behaviour since the latter is outside the scope of this PR.